tensorflow/compiler/mlir/hlo/tests/Dialect/mhlo/ops.mlir

```diff
@@ -965,7 +965,7 @@ func.func @rng_bit_generator(%arg0: tensor<2xui64>) -> (tensor<2xui64>, tensor<1
   %4 = mhlo.constant dense<[10, 12]> : tensor<2xui64>
   %0 = mhlo.constant dense<[10, 12]> : tensor<2xi32>
   %1 = mhlo.constant dense<3> : tensor<i32>
-  %2, %3 = "mhlo.rng_bit_generator"(%4) {rng_algorithm = 0 : i32} : (tensor<2xui64>) -> (tensor<2xui64>, tensor<10x12xui32>)
+  %2, %3 = "mhlo.rng_bit_generator"(%4) {rng_algorithm = #mhlo<"algorithm DEFAULT">} : (tensor<2xui64>) -> (tensor<2xui64>, tensor<10x12xui32>)
```

Comment:
I think` #mhlo.rng<DEFAULT>` should work here then

Reply:
When I replace  #mhlo<"algorithm DEFAULT"> with #mhlo.rng<DEFAULT> It seem didn't work well.